docs/theory/force_calibration/active.rst

```diff
@@ -91,3 +91,94 @@ hydrodynamically correct model for the peak:
+For microfluidic flow, this value is typically much smaller than `1`.
+
+In this limit, the Navier-Stokes equation describing fluid flow reduces to the following expressions:
+
+.. math::
+
+    \begin{align}
+    -\nabla \cdot ( \nabla v + pI ) & = 0\\
+    \nabla \cdot v & = 0
+    \end{align}
+
+Here :math:`p` is the pressure and :math:`v` is the fluid velocity.
+Creeping flow is far removed from every day intuition as it equilibrates instantaneously.
+The advantage of this is that for sufficiently low frequencies, the correction factor can be based on the correction factor one would obtain for a steady state constant flow.
+
+For two beads aligned in the flow direction, we can use the analytical solution presented in :cite:`stimson1926motion`.
+This model uses symmetry considerations to solve the creeping flow problem for two solid spheres moving at a constant velocity parallel to their line of centers.
+We denote the correction factor obtained from this model as :math:`c_{\|}`.
+This correction factor is given by the ratio of the drag coefficient by the drag coefficient one would expected from a single bead in creeping flow (:math:`3 \pi \mu d v`).
+
+For beads aligned perpendicular to the flow direction, we use a model from :cite:`goldman1966slow`, which we denote as :math:`c_{\perp}`.
+
+Considering the linearity of the equations that describe creeping flow :cite:`goldman1966slow`, we can combine these two solutions by decomposing the incoming velocity (in the direction :math:`\vec{e}_{osc}`) into a velocity perpendicular to the bead-to-bead axis :math:`\vec{e}_{\perp}` and a velocity component aligned with the bead-to-bead axis :math:`\vec{e}_{\|}`.
+
+.. math::
+
+    \begin{align}
+    v_{\|} & = (\vec{e}_{\|} \cdot\vec{e}_{osc}) c_{\|}\\
+    v_{\perp} & = (\vec{e}_{\perp} \cdot \vec{e}_{osc})  c_{\perp}
+    \end{align}
+
+This provides us with contributions for each of those axes, but we still need to project this back to the oscillation axis (since this is where we measure our amplitude).
+We can calculate our desired hydrodynamic correction factor as:
+
+.. math::
+
+    c_{total} = v_{\|} (\vec{e}_{\|} \cdot \vec{e}_{osc}) + v_{\perp} (\vec{e}_{\perp} \cdot \vec{e}_{osc})
+
+The response of this combined model for equally sized beads can be calculated as follows::
+
+    diameter = 1.0
+    l_d = np.arange(1.01, 8, 0.1) * diameter
+    zeros = np.zeros(l_d.shape)
+    plt.plot(l_d, lk.coupling_correction_2d(l_d, zeros, diameter, is_y_oscillation=False), label="horizontal alignment [Stimson et al]")
+    plt.plot(l_d, lk.coupling_correction_2d(zeros, l_d, diameter, is_y_oscillation=False), label="vertical alignment [Goldman et al]")
+    plt.plot(l_d, lk.coupling_correction_2d(l_d / np.sqrt(2), l_d / np.sqrt(2), diameter, is_y_oscillation=False), label="diagonal alignment")
+    plt.ylabel('Correction factor [-]')
+    plt.xlabel("l/D [-]")
+    plt.legend()
+
+.. image:: figures/correction_factor.png
```

Comment:
The diameter, D is not defined in the text. And it is also not clear from the text how the correction factor is a function if I/D

Reply:
It was defined in the first section, but it doesn't hurt to redefine it per section, so I'll add it again here. I do note that for consistency, it is probably better to use small case `d` (since we use capital `D` for diffusion and that may end up being unnecessarily confusing).